contracts/test/TestEIP20CoGateway.sol

```diff
@@ -0,0 +1,143 @@
+pragma solidity ^0.5.0;
+
+import "../gateway/EIP20CoGateway.sol";
+
+/**
+ * @title TestEIP20CoGateway contract.
+ *
+ * @notice This is used for testing purpose.
+ */
+contract TestEIP20CoGateway is EIP20CoGateway {
```

Comment:
How do we currently differentiate between `Test...` and `Mock...`?

/cc @deepesh-kn @sarvesh-ost